src/libs/Navigation/AppNavigator/Navigators/OnboardingModalNavigator.tsx

```diff
@@ -17,6 +21,14 @@ const Stack = createStackNavigator<OnboardingModalNavigatorParamList>();
 function OnboardingModalNavigator() {
     const styles = useThemeStyles();
     const {shouldUseNarrowLayout} = useOnboardingLayout();
+    const [hasCompletedGuidedSetupFlow] = useOnyx(ONYXKEYS.NVP_ONBOARDING, {selector: (onboarding) => onboarding?.hasCompletedGuidedSetupFlow ?? true});
+
+    if (hasCompletedGuidedSetupFlow) {
+        Navigation.goBack();
+        Report.navigateToConciergeChat();
+        // eslint-disable-next-line react/jsx-no-useless-fragment
+        return null;
+    }
```

Comment:
```suggestion
    useEffect(() => {
        if (hasCompletedGuidedSetupFlow) {
            Navigation.goBack();
            Report.navigateToConciergeChat();
        }
    }, [hasCompletedGuidedSetupFlow]);

    if (hasCompletedGuidedSetupFlow) {
        return null;
    }
```

Is this the error you found in native apps?
> ERROR  Warning: Cannot update a component (`ForwardRef(BaseNavigationContainer)`) while rendering a different component (`OnboardingModalNavigator`). To locate the bad setState() call inside `OnboardingModalNavigator`, follow the stack trace as described in https://react.dev/link/setstate-in-render

I think this is due to using navigation while rendering. So we need to move it to a `useEffect`.